packages/alt-ergo-lib/alt-ergo-lib.2.5.0/opam

```diff
@@ -0,0 +1,63 @@
+opam-version: "2.0"
+synopsis: "The Alt-Ergo SMT prover library"
+description: """
+This is the core library used in the Alt-Ergo SMT solver.
+
+Alt-Ergo is an automatic theorem prover of mathematical formulas. It was developed at LRI, and is now maintained at OCamlPro.
+
+See more details on http://alt-ergo.ocamlpro.com/"""
+maintainer: ["Alt-Ergo developers <alt-ergo@ocamlpro.com>"]
+authors: ["Alt-Ergo developers <alt-ergo@ocamlpro.com>"]
+homepage: "https://alt-ergo.ocamlpro.com/"
+doc: "https://ocamlpro.github.io/alt-ergo"
+bug-reports: "https://github.com/OCamlPro/alt-ergo/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune" {>= "3.0"}
+  "dune-build-info"
+  "dolmen" {>= "0.9"}
+  "dolmen_type" {>= "0.9"}
+  "dolmen_loop" {>= "0.9"}
+  "ocplib-simplex" {>= "0.5"}
+  "zarith" {>= "1.4"}
+  "seq"
+  "fmt"
+  "stdlib-shims"
+  "ppx_blob" {>= "0.3"}
```

Comment:
I'm unsure which version to require but I'm guessing the first version to have moved to ppxlib is probably a good bet:
```
#=== ERROR while compiling alt-ergo-lib.2.5.0 =================================#
# context              2.2.0~alpha3~dev | linux/x86_64 | ocaml-base-compiler.4.08.1 | file:///home/opam/opam-repository
# path                 ~/.opam/4.08/.opam-switch/build/alt-ergo-lib.2.5.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p alt-ergo-lib -j 31 --promote-install-files=false @install
# exit-code            1
# env-file             ~/.opam/log/alt-ergo-lib-7-5588cb.env
# output-file          ~/.opam/log/alt-ergo-lib-7-5588cb.out
### output ###
# File "/home/opam/.opam/4.08/lib/ppx_blob/ppx_blob.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.08/lib/ppx_derivers/ppx_derivers.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "src/lib/dune", line 28, characters 19-27:
# 28 |   (preprocess (pps ppx_blob))
#                         ^^^^^^^^
# Error: Ppx dependency on a non-ppx library "ppx_blob". If "ppx_blob" is in
# fact a ppx rewriter library, it should have (kind ppx_rewriter) in its dune
# file.
# -> required by library "alt-ergo-lib" in _build/default/src/lib
# -> required by _build/default/META.alt-ergo-lib
# -> required by _build/install/default/lib/alt-ergo-lib/META
# -> required by _build/default/alt-ergo-lib.install
# -> required by alias install
# File "src/lib/dune", line 28, characters 14-28:
# 28 |   (preprocess (pps ppx_blob))
#                    ^^^^^^^^^^^^^^
# (cd _build/default && .ppx/c3461e88ae1503d575d582eb5874e916/ppx.exe --cookie 'library-name="AltErgoLib"' -o src/lib/reasoners/bitv.pp.ml --impl src/lib/reasoners/bitv.ml --dump-ast)
# Fatal error: exception File "src/lib/reasoners/bitv.ml", line 225, characters 6-10: let operators are not supported before OCaml 4.08
# (cd _build/default && .ppx/c3461e88ae1503d575d582eb5874e916/ppx.exe --cookie 'library-name="AltErgoLib"' -o src/lib/util/theories.pp.ml --impl src/lib/util/theories.ml --dump-ast)
# File "src/lib/util/theories.ml", line 38, characters 18-39:
# 38 |   | Fpa -> [%blob "src/preludes/fpa.ae"]
#                        ^^^^^^^^^^^^^^^^^^^^^
# Error: [%blob] could not find or load file /workspace_root/src/lib/util/src/preludes/fpa.ae
```
```suggestion
  "ppx_blob" {>= "0.7.2"}
```